electron/renderer/src/components/Webviews.js

```diff
@@ -63,7 +70,7 @@ class Webviews extends Component {
   }
 
   _onUnreadCountUpdated(accountId, unreadCount) {
-    this.props.updateAccountBadgeCount(account.id, unreadCount);
+    this.props.updateAccountBadgeCount(accountId, unreadCount);
```

Comment:
I am not sure if and why this even worked ...

Reply:
It did not. Was broken since https://github.com/wireapp/wire-desktop/pull/1162